src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -13,171 +13,129 @@
 ~    See the License for the specific language governing permissions and
 ~    limitations under the License.
 -->
-<div *ngIf="isModalShown">
-  <div [config]="{ show: true }" (onHidden)="hideModal()" bsModal #registerWorkflowModal="bs-modal" class="modal fade" tabindex="-1"
-  role="dialog" aria-hidden="true">
-
-  <div class="modal-dialog">
-    <div class="modal-content">
-      <div class="modal-header">
-        <h4 class="modal-title">Register Workflow</h4>
+<h4 mat-dialog-title>Register Workflow</h4>
+<div mat-dialog-content>
+  <div class="alert alert-danger alert-dismissable" role="alert" *ngIf="workflowRegisterError">
+    <button type="button" class="close" data-dismiss="alert" (click)="clearWorkflowRegisterError()">
+      &times;
+    </button>
+    <p>
+      <span class="glyphicon glyphicon-warning-sign"></span>
+      {{workflowRegisterError.message}}
+    </p>
+    <p class="error-output">{{workflowRegisterError.errorDetails}}</p>
+  </div>
+  <div *ngIf="refreshMessage; else placeholder">
+    <mat-progress-bar mode="indeterminate"></mat-progress-bar>
+  </div>
+  <ng-template #placeholder>
+    <div class="pt-2"></div>
+  </ng-template>
+  <mat-horizontal-stepper #stepper>
+    <ng-template matStepperIcon="edit">
+      <mat-icon>done</mat-icon>
+    </ng-template>
+    <mat-step>
+      <ng-template matStepLabel>Workflow storage type</ng-template>
+      <p>
+        You can write your CWL/WDL and store it on Dockstore.org, or you can register a CWL/WDL/NextFlow on an existing third party repository, such
```

Comment:
`... you can register a CWL/WDL/NextFlow from an ...`

Reply:
Nextflow is blocked for now for hosted workflows.

Reply:
That too, I more meant that we should say that the content is "from" a repository rather than "on"
But that's important to note too